sdk/python/feast/registry.py

```diff
@@ -21,6 +21,8 @@
 from typing import List, Optional
 from urllib.parse import urlparse
 
+from google.cloud.exceptions import NotFound
```

Comment:
Should this dependency be eagerly loaded? What about teams that don't use GCP?

Reply:
Good catch - I'll place this import statement in the method.